base/exports.jl

```diff
@@ -453,6 +453,7 @@ export
     sum!,
     sum,
     to_indices,
+    unsafe_freeze!,
```

Comment:
Erroneous export?

Reply:
Left over from when `ImmutableBuffer` was part of the PR.